<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnValider">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnnuler">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="DHCP">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>DHCP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="STATIC">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>STATIC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Carte reseau</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="STATICIP">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>STATIC IP</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>57</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IPV4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>57</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Masque</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Passerelle</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ip">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_masque">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>DNS</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>57</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prefere</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>57</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Auxiliaire</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUI_LAN_CONFIGURATION">
    <property name="title">
     <string>UI LAN CONFIGURATION</string>
    </property>
   </widget>
   <addaction name="menuUI_LAN_CONFIGURATION"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
